config/projects/datadog-agent.rb

```diff
@@ -1,5 +1,5 @@
 name 'datadog-agent'
-maintainer 'Datadog Packages <package@datadoghq.com>'
+maintainer 'Etienne Omnitests <etienne.lafarge@datadoghq.com>'
```

Comment:
Yep I changed the user to be able to use my Test key on Travis. I reverted to the old value :)